src/components/Form/Form.js

```diff
@@ -223,6 +223,7 @@ class Form extends Component {
                                             NewState[title][name] = id;
                                             break;
```

Comment:
Remove unnecessary `break` here.

Reply:
Eslinter was giving a warning for this lol

Reply:
You have a break on the next line right outside the `if` statement, so an extra break inside the `if` is unnecessary